ui/tests/integration/components/secret-engine/configure-create-edit-test.js

```diff
@@ -0,0 +1,805 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
```

Comment:
This is the "new" component test. A couple of things:

-I did my best to run for loops for the tests that made sense to run on each wif engine.
-I did my best to test once if that tests wasn't dependent on engine type.
- And I did my best to cover all engine specific things.
All that to say, I did my best, but my brain is fried after running through this so many times. If you catch an efficiency, please point it out.